src/components/DocumentListItem/DocumentListItem.js

```diff
@@ -11,14 +10,16 @@ import {
   Row,
   Col
 } from "reactstrap";
+import { documentType } from "../../types";
 
 function DocumentListItem(props) {
-  let [expanded, setExpanded] = useState(props.expanded);
+  let [expanded, setExpanded] = useState(false);
```

Comment:
I see no need to pass the initial expanded state down on props, so defaulting to false to simplify.

Reply:
Left to be handled at same time as other expanded stuff